src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/bespoke/JurySummoningMonitorReportsITest.java

```diff
@@ -261,52 +260,52 @@ void viewByCourts() {
                     "date_from", AbstractReportResponse.DataTypeValue.builder()
                         .displayName("Date from")
                         .dataType(LOCAL_DATE)
-                        .value("2024-01-01")
+                        .value("2024-08-01")
                         .build(),
                     "date_to", AbstractReportResponse.DataTypeValue.builder()
                         .displayName("Date to")
                         .dataType(LOCAL_DATE)
-                        .value("2024-05-01")
+                        .value("2024-08-07")
                         .build(),
                     "report_created", AbstractReportResponse.DataTypeValue.builder()
                         .displayName("Report created")
                         .dataType(LOCAL_DATE)
                         .value(LocalDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE))
                         .build()))
-                .totalJurorsNeeded(0)
-                .bureauDeferralsIncluded(0)
-                .bureauToSupply(0)
-                .initiallySummoned(0)
-                .ratio(0.0)
-                .additionalSummonsIssued(0)
-                .reminderLettersIssued(0)
-                .totalConfirmedJurors(0)
-                .deferralsRefused(0)
-                .excusalsRefused(0)
-                .totalUnavailable(0)
-                .nonResponded(0)
-                .undeliverable(0)
-                .awaitingInformation(0)
-                .disqualifiedPoliceCheck(0)
-                .disqualifiedOther(0)
-                .deferred(0)
-                .postponed(0)
-                .excused(0)
+                .totalJurorsNeeded(17)
+                .bureauDeferralsIncluded(1)
+                .bureauToSupply(16)
+                .initiallySummoned(16)
+                .ratio(1.0)
```

Comment:
Would be good to have a ratio with a decimal to ensure formatting is okay (Such as 0.75)

Reply:
Have updated to give ratio below 1